cmd/config_git_set.go

```diff
@@ -59,6 +59,25 @@ func NewConfigGitSetCmd(newClient ClientFactory) *cobra.Command {
 		"Specify a custom builder image for use by the builder other than its default. ($FUNC_BUILDER_IMAGE)")
 	cmd.Flags().StringP("image", "i", f.Image, "Full image name in the form [registry]/[namespace]/[name]:[tag]@[digest]. This option takes precedence over --registry. Specifying digest is optional, but if it is given, 'build' and 'push' phases are disabled. (Env: $FUNC_IMAGE)")
 
+	// Git related Flags:
+	cmd.Flags().StringP("git-url", "g", "",
+		"Repo url to push the code to be built (Env: $FUNC_GIT_URL)")
```

Comment:
Is code pushed to the repo?
```suggestion
		"Repository url containing the function to build (Env: $FUNC_GIT_URL)")
```

Reply:
Thanks, that's much better. I copy pasted this from `cmd/deploy.go` without proper reading, I fixed it there as well.